cmd/ttn-lw-cli/commands/gateways.go

```diff
@@ -377,6 +408,7 @@ func init() {
 	gatewaysCreateCommand.Flags().AddFlagSet(setGatewayFlags)
 	gatewaysCreateCommand.Flags().AddFlagSet(setGatewayAntennaFlags)
 	gatewaysCreateCommand.Flags().AddFlagSet(attributesFlags())
+	gatewaysCreateCommand.Flags().Bool("defaults", true, "configure gateway with default gateway server address")
```

Comment:
@johanstokking don't you think that while we're at it, we should make the defaults similar to end devices? The default options are, same as with the end devices, options that most users would want on almost all devices, such as `enforce-duty-cycle`.

Reply:
@htdvisser I believe that for end devices the defaults are also set only for [ns/as](https://github.com/TheThingsNetwork/lorawan-stack/blob/92a8b210b6516228785e35239ed7e1e2dc455e45/cmd/ttn-lw-cli/commands/end_devices.go#L310) and [js](https://github.com/TheThingsNetwork/lorawan-stack/blob/92a8b210b6516228785e35239ed7e1e2dc455e45/cmd/ttn-lw-cli/commands/end_devices.go#L359), correct me if I'm wrong

Reply:
@htdvisser agreed. I would propose;

- `auto_update` on
- `enforce_duty_cycle` on
- `status_public` on
- `location_public` on

Reply:
The comment for this flag needs to be updated.

Reply:
Also Gateway Server is written in title case